graylog2-server/src/test/java/org/graylog/testing/mongodb/MongoDBContainer.java

```diff
@@ -30,7 +30,7 @@
  */
 public class MongoDBContainer extends GenericContainer<MongoDBContainer> {
     public static final String DEFAULT_IMAGE = "mongo";
-    public static final String DEFAULT_VERSION = "3.6";
+    public static final String DEFAULT_VERSION = "4.4";
```

Comment:
Can we make this `4.0`? We still support MongoDB 4.0 so we should use the oldest supported version, I think.

Reply:
Fine with me. Testing against newer versions also has its benefits though.
For the leader election, we do this: https://github.com/Graylog2/graylog2-server/pull/11586/commits/94adff71728e30bb5f54a7a106b190efe0bc4ca4